### PR TITLE
FAI-11591 | Organizations stream (source + converter)

### PR DESCRIPTION
**Pull request by dbruno21**
## Description

Organizations stream (source + converter)

## Type of change
- [ ] Bug fix
- [X] New feature
- [ ] Breaking change

### CODE REVIEW

File: destinations/airbyte-faros-destination/src/converters/github/faros_organizations.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+import {DestinationModel, DestinationRecord, StreamContext} from '../converter';
+import {GitHubConverter} from './common';
+
+export class FarosOrganizations extends GitHubConverter {
+  readonly destinationModels: ReadonlyArray<DestinationModel> = [
+    'vcs_Organization',
+  ];
+
+  async convert(
```
Comment (chalenge) [inline destinations/airbyte-faros-destination/src/converters/github/faros_organizations.ts:14]: let's re-use the current CE `Organizations` stream, see [example](https://github.com/faros-ai/airbyte-connectors/blob/main/destinations/airbyte-faros-destination/src/converters/jira/faros_project_versions.ts#L7-L10)

File: sources/github-source/resources/schemas/farosOrganizations.json [hunk 2]
```diff
@@ -0,0 +1,438 @@
+{
+  "title": "Organization Full",
```
Comment (chalenge) [inline sources/github-source/resources/schemas/farosOrganizations.json:2]: Organizations

File: sources/github-source/resources/schemas/farosOrganizations.json [hunk 3]
```diff
@@ -0,0 +1,438 @@
...
+        "github"
+      ]
+    },
+    "id": {
+      "type": "integer",
+      "examples": [
+        1
+      ]
+    },
+    "node_id": {
```
Comment (chalenge) [inline sources/github-source/resources/schemas/farosOrganizations.json:18]: can we just keep the properties we need

File: sources/github-source/src/github.ts [hunk 4]
```diff
@@ -56,15 +56,15 @@ export class GitHub {
   }
 
   @Memoize()
-  async getOrganizations(): Promise<ReadonlyArray<Organization>> {
-    const orgs: Organization[] = [];
-    for await (const org of this.getOrganizationsIterator()) {
+  async getAccessibleOrganizations(): Promise<ReadonlyArray<string>> {
```
Comment (chalenge) [inline sources/github-source/src/github.ts:59]: can we just keep it as `getOrganizations`

**Review by chalenge — COMMENTED** [review]
Can you re-use the CE Organizations stream

**Comment by dbruno21** [comment]
@chalenge 
PTAL

**Review by ypc-faros — APPROVED** [review]

**Pull request merged**